modules/explorer/info.go

```diff
@@ -1,8 +1,12 @@
 package explorer
 
 import (
+	"errors"
+	"github.com/NebulousLabs/Sia/build"
+	"github.com/NebulousLabs/Sia/encoding"
 	"github.com/NebulousLabs/Sia/modules"
 	"github.com/NebulousLabs/Sia/types"
+	"github.com/NebulousLabs/bolt"
```

Comment:
convention is to separate imports like so:
```go
 import (
	"errors"

	"github.com/NebulousLabs/Sia/build"
	"github.com/NebulousLabs/Sia/encoding"
 	"github.com/NebulousLabs/Sia/modules"
 	"github.com/NebulousLabs/Sia/types"

	"github.com/NebulousLabs/bolt"
)
```
i.e. first stdlib, then internal, then external